app/mishki-index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="css/style.min.css">
+  <title>Mishka</title>
+</head>
+<body>
+  <header class="header">
+    <nav class="nav">
+      <div class="nav--wrapper">
+        <img class="nav__logo logo" src="/app/imajes/jpg/mishka-logo-mobile.svg copy.png" alt="Мишка">
+        <button class="main-nav__togle" type="button"><?xml version="1.0" encoding="iso-8859-1"?>
+          <svg class="svg__icon" version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+             width="35px" height="30px" viewBox="0 0 612 612" style="enable-background:new 0 255 255 255;" xml:space="preserve">
```

Comment:
```suggestion
          <svg 
              class="svg__icon" 
              version="1.1" 
              id="Capa_1" 
              xmlns="http://www.w3.org/2000/svg" 
              xmlns:xlink="http://www.w3.org/1999/xlink" 
              x="0px" 
              y="0px"
              width="35px" 
              height="30px" 
              viewBox="0 0 612 612" 
              style="enable-background:new 0 255 255 255;" 
              xml:space="preserve"
          >
```

вот так ведь гораздо легче прочитать, не правда ли?)